src/components/backup/RestoreCloudStep.tsx

```diff
@@ -119,6 +124,7 @@ export default function RestoreCloudStep() {
     const fetchPasswordIfPossible = async () => {
       const pwd = await getLocalBackupPassword();
       if (pwd) {
+        backupsStore.getState().setStoredPassword(pwd);
```

Comment:
by setting this here, we can prevent the need to call `getLocalBackupPassword` later on (which prompts faceID)